packages/scss/src/components/button/mods.scss

```diff
@@ -68,6 +72,18 @@
 	}
 }
 
+@mixin onlyIcon {
+	padding: var(--spacings-XS);
```

Comment:
Wouldn't a CSS variable be nice here?

Reply:
Ah, yes, and we already had one. 
Fixed.